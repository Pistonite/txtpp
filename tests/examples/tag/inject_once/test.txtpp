TXTPP#tag tag
TXTPP#include include
Inject the tag
Should not inject the tag again