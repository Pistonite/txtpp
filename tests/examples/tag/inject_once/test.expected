Inject the include content
Should not inject the tag again
